web/src/ColorGenerate.js

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 The Casdoor Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const path = require("path");
```

Comment:
Use `import` instead of `require`

Reply:
![image](https://user-images.githubusercontent.com/41513919/188300846-5af4c95c-7322-44aa-b4c6-7babf26ece35.png)

This file it's just used to generate runtime `.less`(run the file when `yarn build` or `yarn start`), not a node module, do we must use `import` here too?

Reply:
We are not Node.js project, `require` is forbidden